cmd/oras/root/login.go

```diff
@@ -73,6 +75,8 @@ Example - Log in with username and password in an interactive terminal and no TL
 			return runLogin(cmd, opts)
 		},
 	}
+	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", true, "print status output for unnamed blobs")
```

Comment:
nit: Although the description is hidden, how about making the description simply "verbose output"? The original description looks a bit misleading in code.

Reply:
Thanks for catching! It should be "verbose output", I forgot to change the description when pasting.